### PR TITLE
When to cloud

### PR DESCRIPTION
**Pull request by alexishunzinger**
New page added to the cookbook is now under progress.

Ideas are listed in bullet form, but draft will be written.

### CODE REVIEW

**Comment by jules32** [comment]
Hi @alexishunzinger ! Would you be ok merging this into the Cookbook, even in its current bulleted form? We're preparing to post a blog post about the Champions Cohort and it would be nice to link to it if you're ready

**Comment by alexishunzinger** [comment]
Yes @jules32, we can push to main for that purpose.

File: when-to-cloud.qmd [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: "When To Cloud"
+subtitle: "Is cloud access and analysis for you?" 
+date: last-modified
+author: "NASA Openscapes Team"
+citation_url: https://nasa-openscapes.github.io/earthdata-cloud-cookbook/when-to-cloud.html
+slug: index
+---
+
+## When To Cloud
```
Comment (stefaniebutland) [inline when-to-cloud.qmd:10]:
```suggestion
```

File: when-to-cloud.qmd [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- listing questions from Andy's slides
+    - a bit of discussion + examples for each question
+    - concrete examples accomapnying each question
+    - "I have a TB of storage on my laptop...." storage scenarios
+    - (see this from Ryan Abernathy: https://medium.com/pangeo/closed-platforms-vs-open-architectures-for-cloud-native-earth-system-analytics-1ad88708ebb6)
+    - include a question on actions that are already services (subsetting, etc.)
+
+
+### Challenges
+It is important to be aware of the drawbacks and challenges associated with working in the cloud. Common feedback from early cloud adopters are summarized here:
```
Comment (stefaniebutland) [inline when-to-cloud.qmd:31]:
```suggestion
It is important to be aware of the drawbacks and challenges associated with working in the cloud. Common feedback from early cloud adopters are summarized here:

```

File: when-to-cloud.qmd [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    - feedback from Champions/workshop participants
+    - (see Alexis ESIp poster)
+    - (this could be its own section on website?)
+
+### Considerations
+We are now accustomed to living in a highly digital world, separated from physical reminders of the services we use. No longer do we access documents from a row of filing cabinets, we now store them in cloud-based archives (e.g. Google Docs). We run code on high-performance computing clusters, removed from the whirring and warmth generated by servers that live away from our desks. The following are considerations for using cloud-based resources:
```
Comment (stefaniebutland) [inline when-to-cloud.qmd:37]:
```suggestion
We are now accustomed to living in a highly digital world, separated from physical reminders of the services we use. No longer do we access documents from a row of filing cabinets, we now store them in cloud-based archives (e.g. Google Docs). We run code on high-performance computing clusters, removed from the whirring and warmth generated by servers that live away from our desks. The following are considerations for using cloud-based resources:

```

Comment (stefaniebutland) [reply when-to-cloud.qmd:37]: I've added a hard return after text and before bullet list, thinking that is required for bullets to appear, not folded into a paragraph

File: when-to-cloud.qmd [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+subtitle: "Is cloud access and analysis for you?" 
+date: last-modified
+author: "NASA Openscapes Team"
+citation_url: https://nasa-openscapes.github.io/earthdata-cloud-cookbook/when-to-cloud.html
+slug: index
+---
+
+## When To Cloud
+
+- opening sentence(s) with link to homepage/Welcome on cloud
```
Comment (stefaniebutland) [inline when-to-cloud.qmd:12]:
```suggestion
Cloud migration can often have a steep learning curve and feel overwhelming. There are times when Cloud is effective and times when the download model may still be more appropriate. Here we aim to help people decide what's best for their use cases.
```

**Review by stefaniebutland — COMMENTED** [review]
@alexishunzinger Thank you for adding this section! I've made some minor commit-suggestions. Julie and I will likely merge this today so we can link to it from the 2023-nasa-champions blog post to be published Aug 1.

**Review by stefaniebutland — APPROVED** [review]

**Comment by stefaniebutland** [comment]
@alexishunzinger We've merged your PR (thank you!). Do you have specific links to your ESIP 2023 poster, or other slides summarizing user feedback, that you'd like to add?

**Pull request merged**